docs/src/constraints.md

```diff
@@ -603,22 +603,38 @@ The constraint `con` is now equivalent to `2x <= 2`.
     `const_term * x` is bilinear. Fixed variables are not replaced with
     constants when communicating the problem to a solver.
 
+Another option is to use [`add_to_function_constant`](@ref). The constant given
+is added to the function of a `func`-in-`set` constraint. As the constant is
+*added* and not *set*, the user should not worry about whether the constant is
```

Comment:
"As the constant is ...": This sentence may be more confusing than helpful. The definition of the function is sufficient to not worry about the normalized form.